artifactory/services/utils/multipartupload.go

```diff
@@ -262,8 +263,9 @@ func (mu *MultipartUpload) uploadPart(logMsgPrefix, localPath string, fileSize,
 }
 
 func (mu *MultipartUpload) createMultipartUpload(repoKey, repoPath string, partSize int64) (token string, err error) {
-	url := fmt.Sprintf("%s%screate?repoKey=%s&repoPath=%s&partSizeMB=%d", mu.artifactoryUrl, uploadsApi, repoKey, repoPath, partSize/SizeMiB)
-	resp, body, err := mu.client.SendPost(url, []byte{}, mu.httpClientsDetails)
+	requestUrl := fmt.Sprintf("%s%screate?repoKey=%s&repoPath=%s&partSizeMB=%d", mu.artifactoryUrl, uploadsApi, url.QueryEscape(repoKey), url.QueryEscape(repoPath), partSize/SizeMiB)
+	log.Debug("Request final url", requestUrl)
```

Comment:
```suggestion
```

There is already a debug log in the HTTP client: https://github.com/jfrog/jfrog-client-go/blob/v1.43.1/http/httpclient/client.go#L175